README.md

```diff
@@ -6,12 +6,14 @@ Currently Clarity supports Android and iOS including popular cross-platform fram
 
 ## Supported Platforms and Frameworks
 
-|           | Android | iOS | Notes |
-|-----------|---------|-----|-------|
-| Native | Full Support | Basic Support | - For iOS, only **UIKit** views are supported. **SwiftUI** support is a work in progress. |
-| React Native | Full Support | Basic Support | - React Native is only enabled behind experimental flag `enableIOS_experimental`. |
-| Cordova | Full Support | No Support | |
-| Ionic | Full Support | No Support | |
+Platform       | Android | iOS     |
+-------------- | :-----: | :-----: |
+All Frameworks | Requires Android API level 29 to 34 | Requires iOS version 15 to 18 |
```

Comment:
Let's link msdocs.

Reply:
@ibradwan You mean removing all content in this README except for a link to Learn, right?